### ISSUE TITLE
Build failed - Ubuntu 16.04

### ISSUE THREAD

**Issue by marcinch18** [EOSIO/eos#2973]
I was building on fresh Ubuntu 16.04 and got this error message:

![image](https://user-images.githubusercontent.com/9887252/39950357-5e978450-5578-11e8-9df9-7c0a36607c1d.png)

**Comment by jgiszczak** [comment]
You're experiencing a network error of some kind, and your package lists aren't updating properly.  Hopefully it's only transient and you can just retry.  Otherwise, you'll need to discover why your installation can't access the network and its repositories successfully.

**Comment by tbfleming** [comment]
I hit the same thing. Not a network issue. Instead, it's trying to install a package named ```" clang-4.0   lldb-4.0   ..."```. i.e. a shell quoting issue.

**Comment by BinaryFocus** [comment]
same problem here. I'm following exactly the same process used to compile dawn 3.0.0 .... launch new AWS Ubuntu image:

 git clone https://github.com/EOSIO/eos.git --recursive
 cd eos/
 git checkout tags/dawn-v4.0.0
 ./eosio_build.sh

```
        Installing dependencies

Hit:1 http://us-east-1.ec2.archive.ubuntu.com/ubuntu xenial InRelease
Get:2 http://us-east-1.ec2.archive.ubuntu.com/ubuntu xenial-updates InRelease [109 kB]
Get:3 http://us-east-1.ec2.archive.ubuntu.com/ubuntu xenial-backports InRelease [107 kB]
Hit:4 http://security.ubuntu.com/ubuntu xenial-security InRelease
Fetched 216 kB in 0s (874 kB/s)
Reading package lists... Done
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package  clang-4.0   ..........
```

**Comment by wuyahuang** [comment]
The build script works after I install those packages manually.

**Comment by pacificcode** [comment]
@wuyahuang @BinaryFocus apt-get doesn't like the literal quotes for the dependency list.
if ! sudo apt-get -y install "${DEP}"
You can unquote the variable "${DEP}" if you need to.
Ill have a PR in for this on Monday.

**Comment by pacificcode** [comment]
This issue was addressed by PR #2976.

**Comment by AaronFaltesek** [comment]
ran 
git pull 
git submodule update --init --recursive 

and after https://github.com/EOSIO/eos/commit/444fed0418bed90c133ad99632d7f630ed0cbf60 this did the trick for me.

I was running into the same issue.
